.circleci/config.yml

```diff
@@ -88,16 +88,16 @@ jobs:
           name: Install integration test dependencies
           command: |
             # Install kubectl
-            curl --retry 5 -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.8.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+            curl --retry 5 -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
             # Install minikube
-            curl --retry 5 -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+            curl --retry 5 -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
             # Install yq
             curl --retry 5 -Lo yq https://github.com/mikefarah/yq/releases/download/1.14.0/yq_linux_amd64 && chmod +x yq && sudo mv yq /usr/local/bin/
       - &start_minikube
         run:
           name: Start minikube
           command: |
-            sudo minikube start --vm-driver=none
+            sudo minikube start --vm-driver=none start --kubernetes-version v1.9.0
```

Comment:
Is there a reason to add the `--kubernetes-version` argument? 0.25.2 seems to ship with 1.9.4, which sounds fine.

Reply:
I think if we pin against a certain version, then we know what exact version failed on us. And we avoid problems like I ran into yesterday when I wanted to use a kubectl feature that was not available yet, but was not aware we are starting a 1.8.4 cluster.

Reply:
There are two `start`s on that line!

I'd use the latest release of that branch (1.9.4), but probably not an issue.

Reply:
aaaah :D copy paste mistake, good catch!

sure 1.9.4 sgtm, adjusted 👍 Really don't have a preference, although there were some memory leaks problems in 1.9.0 version IIRC, but I doubt it would effect our tests.